pennylane/ops/op_math/composite.py

```diff
@@ -55,16 +55,16 @@ def __init__(
         if len(operands) < 2:
             raise ValueError(f"Require at least two operators to combine; got {len(operands)}")
 
+        if not do_queue:
+            raise ValueError(f"do_queue=False is not supported for {self.name}")
```

Comment:
Oh actually one thought, how about we give users a suggested path? Like `stop_recording`?